DRAFT_COROLLARY.md

```diff
@@ -213,11 +214,12 @@ template <class U>
 constexpr indirect& operator=(U&& u);
 ```
 
-A. _Constraints_: `is_constructible<T, U>` is true.
-  `is_assignable<T,U>` is true.
+A. _Constraints_: `is_constructible<T, U>` is true. `is_assignable<T,U>` is
+  true.
 
 B. _Mandates_: `T` is a complete type.
 
+
 C. _Effects_: If `this` is valueless then equivalent to
    `*this = indirect<T>(std::forward<U>(u))`.
```

Comment:
This uses a default constructed allocator, which is not listed as a constraint of calling this assignment operator. Should it be `*this = indirect(allocator_arg, alloc, std::forward<U>(u));` or should it be constrained to require `is_default_constructible_v<Allocator>`? Or something else?

Reply:
Nicely spotted. Agreed and applied manually.